plone/app/tiles/modifiers.py

```diff
@@ -67,6 +67,8 @@ def manage_addCleanTileAnnotations(self, id, title=None, REQUEST=None):
         REQUEST['RESPONSE'].redirect(self.absolute_url() + '/manage_main')
 
 
+ANNOTATION_KEY_PREFIXES = [ANNOTATIONS_KEY_PREFIX,  # plone.tiles default
+                           'plone.tiles.scale']  # collective.cover scales
```

Comment:
do you see any way to improve it at collective.cover implementation,  instead of create a working around here..?